modules/mcagar/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Added
+- Added `promotion` set to `false` as a default setting.
```

Comment:
```suggestion
### Changed
- The new `promotion` option added in TinyMCE 6.2 is set to `false` by default.
```
Added is meant to be new features, since this isn't new then it should probably be either an enhancement or a simple change... I figured the latter made more sense in this usecase 🤷